code/datums/status_effects/buffs.dm

```diff
@@ -450,3 +450,77 @@
 /datum/status_effect/exercised/Destroy()
 	. = ..()
 	STOP_PROCESSING(SSprocessing, src)
+
+//Hippocratic Oath: Applied when the Rod of Asclepius is activated.
+/datum/status_effect/hippocraticOath
+	id = "Hippocratic Oath"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	tick_interval = 25
+	examine_text = "<span class='notice'>SUBJECTPRONOUN seems to have an aura of healing and helpfulness about them.</span>"
+	alert_type = null
+	var/hand = null
+
+/datum/status_effect/hippocraticOath/on_apply()
+	//Makes the user passive, it's in their oath not to harm!
+	owner.add_trait(TRAIT_PACIFISM, "hippocraticOath")
+	var/datum/atom_hud/H = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	H.add_hud_to(owner)
+	return ..()
+
+/datum/status_effect/hippocraticOath/on_remove()
+	owner.remove_trait(TRAIT_PACIFISM, "hippocraticOath")
+	var/datum/atom_hud/H = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	H.remove_hud_from(owner)
+
+/datum/status_effect/hippocraticOath/tick()
+	//Checks to make sure the rod is still in their hand
+	if(iscarbon(owner))
+		var/mob/living/carbon/itemUser = owner
+		var/obj/item/heldItem = itemUser.get_item_for_held_index(hand)
+		if(heldItem == null || heldItem.type != /obj/item/rod_of_asclepius)
+			var/obj/item/rod_of_asclepius/newRod = new /obj/item/rod_of_asclepius(itemUser.loc)
+			newRod.flags_1 = NODROP_1 | DROPDEL_1
+			newRod.desc = "A short wooden rod with a mystical snake inseparably gripping itself and the rod to your forearm. It flows with a healing energy that disperses amongst yourself and those around you. "
+			newRod.icon_state = "asclepius_active"
+			newRod.activated = TRUE
+			if(!itemUser.has_hand_for_held_index(hand))
+				//If user does not have the corresponding hand anymore, give them one and return the rod to their hand
+				if(((hand % 2) == 0))
+					var/obj/item/bodypart/L = itemUser.newBodyPart("r_arm", FALSE, FALSE)
+					L.attach_limb(itemUser)
+					itemUser.put_in_r_hand(newRod)
+				else
+					var/obj/item/bodypart/L = itemUser.newBodyPart("l_arm", FALSE, FALSE)
+					L.attach_limb(itemUser)
+					itemUser.put_in_l_hand(newRod)
+				to_chat(itemUser, "<span class='notice'>Your arm suddenly grows back with the Rod of Asclepius still attached!</span>")
+			else
+				//Otherwise get rid of whatever else is in their hand and return the rod to said hand
+				itemUser.dropItemToGround(itemUser.get_item_for_held_index(hand))
+				if(((hand % 2) == 0))
+					itemUser.put_in_r_hand(newRod)
+				else
+					itemUser.put_in_l_hand(newRod)
+				to_chat(itemUser, "<span class='notice'>The Rod of Asclepius suddenly grows back out of your arm!</span>")
```

Comment:
This just seems cheap. It prevents the individual from gaming it for free heals and then dropping it by REMOVING THEIR ARM (tedious), but it doesn't stop an attacker from disabling those heals even if they manage to rip the person's arm off.

Reply:
I mean, they can just kill them

Reply:
There's no stat check :^)

Reply:
NOR ARE THE BODIES PAIR THIS WITH SOMEONE WHO USES A DEFIB!!!!!!!

Reply:
Doesn't remove the time constraint though. Of course, strange reagent smoke is a different story...

Reply:
Yeah needs to check for death

Reply:
From my experiences, the damage is usually the limiting factor rather than time.

Reply:
Alright, I have a plan for making it not only end on death, but making death even more unviable for the user